### PR TITLE
Update my PPP

### LINKED ISSUE #174: Update PPP

### PR DESCRIPTION
**Pull request by glyfy**
Resolves #174

### CODE REVIEW

**Review by sanjevi13 — APPROVED** [review]
LGTM.

File: docs/team/glyfy.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Given below are my contributions to the project.
      * Added implementation details of the `findBook` feature.
 
 * **Community**:
-    * PRs reviewed (with non-trivial review comments): [\#165](https://github.com/AY2223S2-CS2103T-T12-3/tp/pull/165)
+    * PRs reviewed (with non-trivial review comments): 
+  [\#165](https://github.com/AY2223S2-CS2103T-T12-3/tp/pull/165),
+  [\#31](https://github.com/AY2223S2-CS2103T-T12-3/tp/pull/31)
```
Comment (sanjevi13) [inline docs/team/glyfy.md:36]: Mismatch with PR number in title

**Review by sanjevi13 — CHANGES_REQUESTED** [review]
Wrong number of PR

**Review by sanjevi13 — APPROVED** [review]
LGTM.

File: docs/team/glyfy.md (hunk 1 above)
Comment (glyfy) [reply docs/team/glyfy.md:36]: Just approve it I cant change the commit message thanks

**Pull request merged**